src/Sarif/Visitors/SarifTransformerUtilities.cs

```diff
@@ -84,6 +87,23 @@ public static class SarifTransformerUtilities
             { "sha-512", AlgorithmKindVersionOne.Sha512 }
         };
 
+        public static IList<TTo> TransformList<TFrom, TTo>(IList<TFrom> fromList, Func<TFrom, TTo> convertFunc)
```

Comment:
I think you just re-invented LINQ `Aggregate` :-)

Reply:
Yeah but mine handles a null list. Couldn't think of a better way.

Reply:
Ah I can use ?
I'll change this...